bin/fin

```diff
@@ -2009,9 +2009,19 @@ ssh_add () {
 # Reports version stats
 stats_ping ()
 {
-	local os_version="$(uname -s)"
-	[[ $DOCKSAL_STATS_OPTOUT == 0 ]] && \
-		curl -kfsL --data "v=1&tid=${DOCKSAL_STATS_TID}&cid=${DOCKSAL_UUID}&t=event&ec=version&ea=${os_version}&el=${FIN_VERSION}" ${DOCKSAL_STATS_URL}
+	[[ "$DOCKSAL_STATS_OPTOUT" != 0 ]] && return # Don't run if the user opted out
```

Comment:
We already have SKIP_VBOX_VERSION_CHECK, should we maybe keep alike options named consistently?
Like SKIP_STATS_GATHERING?